docs/pagination.md

```diff
@@ -0,0 +1,71 @@
+# Pagination
+
+Pagination is supported using the metadata support that is built into Ember Data. The
+metadata from Django REST Framework paginated list views is updated on every request
+to sever.
+
+
+## Retrieving the Metadata
+
+To get a page of records simply run a find request with the `page` query param:
+
+```js
+var result = this.store.find("post", {
+  page: 1
+});
+```
+
+It's possible to override the server-side page size by also including the `page_size`
```

Comment:
It's already possible to use any param name for page size. ED just passes the param to the backend and EDA doesn't change that. (I'll document this as well)

The issue is that if PAGINATE_BY_PARAM is disabled (which is the default), the application doesn't know the page size. Of course front-end ember applications can always hardcode the PAGINATE_BY value but that is just a copy of the back-end DRF setting for the view. It seems more flexible if the serializer sets the page size as metadata so applications can "read" the page size. If the back-end DRF setting changes, the correct page size will be automatically in the metadata. This isn't an issue if an application uses 'page_size' explicitly because the front-end application is in charge of the setting in this case, not the back-end.